seb_openedx/views.py

```diff
@@ -24,10 +25,14 @@ def info_view(request):
     except CalledProcessError:
         git_data = ''
 
+    middlewares = settings.MIDDLEWARE_CLASSES if hasattr(settings, 'MIDDLEWARE_CLASSES') else settings.MIDDLEWARE
```

Comment:
This whole function is cool for debugging, but its very own existence should be defaulted to false. The reasoning being that we don't want to let a potencial fraudster know which exact version of the code will answer to his/her requests.

Reply:
Done (now it works for staff only)